packages/testing/src/apiTestingMocks.ts

```diff
@@ -0,0 +1,111 @@
+import type {
+  APIGatewayProxyEvent,
```

Comment:
Since these are specific to AWS Lambdas, do you think we should plan for future where functions might be azure, google, cloudfront, etc and put these mocks in a `awsLambda` directory?

For example,"I need to mock an AWS event" vs "I want to mock an Azure event"

Reply:
I think we should cross that bridge when we get there! I imagine what event we would use would probably be controlled by a flag in redwood.toml?